packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -521,6 +571,10 @@ function createRootStyles(themeVariables?: ThemeVariables) {
         --wui-color-success-glass-060: rgba(38, 217, 98, 0.6);
         --wui-color-success-glass-080: rgba(38, 217, 98, 0.8);
 
+        --wui-color-success-glass-reown-020: rgba(48, 164, 107, 0.2);
+
+        --wui-color-warning-glass-reown-020: rgba(243, 161, 63, 0.2);
```

Comment:
Since we're using reown colors / backgrounds for the alerts i've included `reown` name here to make it easier.